src/Activation/AcceptEula.tsx

```diff
@@ -41,7 +41,8 @@ const AcceptEula: FunctionComponent = () => {
   }
 
   const linkToEula = async () => {
-    await Promise.resolve()
+    configuration.healthAuthorityEulaUrl &&
```

Comment:
Just curious why the double check on `configuation.healthAuthorityEulaUrl`?

Reply:
That's the thing with encapsulating the method outside, Typescript has no idea that the only place this is being called it's surrounded by a null check 😢

Reply:
Ah I see, dammit Typescript!